Missing vuetify dependency from package.json

Just pulled down your project to take a look, ran npm install, then npm run dev to get the following error.

Looks like vuetify is missing from your package.json file

```
Failed to compile.

./src/main.js
Module not found: Error: Can't resolve 'vuetify' in 'vue-recipes/src'
 @ ./src/main.js 2:0-30
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./src/main.js
```

--- a/package.json
+++ b/package.json
@@ -10,7 +10,8 @@
   },
   "dependencies": {
     "vue": "^2.4.4",
-    "vuedraggable": "^2.14.1"
+    "vuedraggable": "^2.14.1",
+    "vuetify": "^1.1.4"
   },
   "devDependencies": {
     "babel-core": "^6.26.0",

